TypeSystem/Schema/schema.h

```diff
@@ -1115,6 +1117,202 @@ struct LanguageSyntaxImpl<Language::JSON> final {
   // LCOV_EXCL_STOP
 };
 
+template <>
+struct LanguageSyntaxImpl<Language::TypeScript> final {
+  static std::string Header(const std::string&) {
+    // TODO(sompylasar): Implement common C++ and Current types in TypeScript.
+    return (
```

Comment:
Sure, seconding the idea to prod it as is sooner, and make it prettier later. Thanks again!